pkg/apis/config/feature_flags.go

```diff
@@ -113,7 +114,7 @@ type FeatureFlags struct {
 	SendCloudEventsForRuns           bool
 	AwaitSidecarReadiness            bool
 	EnforceNonfalsifiability         string
-	ResourceVerificationMode         string
+	VerificationNoMatchPolicy        string
```

Comment:
Add godoc comment (really all of these should have godoc comments, but might as well start here 🤷 )